pandas/tests/reshape/merge/conftest.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+
+
+@pytest.fixture(params=["left", "right", "inner", "outer"])
+def how(request):
```

Comment:
This exists as `join_type` in `pandas/conftest.py`

Reply:
Oh thx, I've been looking for this. Added